ir_datasets/datasets/trec_fair_2021.py

```diff
@@ -0,0 +1,179 @@
+import json
+import mwparserfromhell
+import codecs
+from typing import NamedTuple, Dict
+import ir_datasets
+from ir_datasets.util import GzipExtract, Cache, Lazy
+from ir_datasets.datasets.base import Dataset, YamlDocumentation, FilteredQueries
+from ir_datasets.formats import BaseQueries, BaseDocs, BaseQrels, TrecQrel
+from ir_datasets.indices import PickleLz4FullStore
+import pyautocorpus
+from itertools import chain
+
+
+_logger = ir_datasets.log.easy()
+
+
+NAME = 'trec_fair_2021'
```

Comment:
Let's use hyphens instead of underscores for the dataset ID. (The python file names will need to replace the hyphens with underscores, though.)